code/iLasso_for_SCINGE.m

```diff
@@ -73,18 +127,34 @@
 w = fit.beta;
 
 % Reformatting the output
-result = zeros(P, L0);
 count = 0; genes = []; areas = [];
 
-for_metrics.Am = Am;
-for_metrics.bm = bm;
-for_metrics.w = w;
+%for_metrics.Am = Am;
+%for_metrics.bm = bm;
+%for_metrics.w = w;
 for_metrics.a0 = fit.a0;
-for_metrics.bic = BIC;
-if isempty(w)
-    result = zeros(P);
-else
-    for i = 1:P
-        result(i, :) = w((i-1)*L0+1:i*L0);
+for_metrics.order = pa;
+for_metrics.params = params;
+p = params.p;
+for ii = 1:size(w,2)
```

Comment:
For each output of `w`, which now includes all values of lambda, collapse all the lags.  `w` is number of regulators X number of lags by number of lambda values.